### PR TITLE
Add basic test for "gettype" function. Check for "unknown type"

### PR DESCRIPTION
**Pull request by yuklia**
User Group: [Kharkiv PHP](https://github.com/phpcommunity/phptestfest.org/pull/70)

### CODE REVIEW

**Review by krakjoe — APPROVED** [review]

**Review by krakjoe — APPROVED** [review]

**Comment by krakjoe** [comment]
I ... don't know why I approved it twice ... Mondays ...

**Review by carusogabriel — APPROVED** [review]
This covers http://gcov.php.net/PHP_HEAD/lcov_html/ext/standard/type.c.gcov.php line 37 :clap:

File: ext/standard/tests/general_functions/gettype_basic.phpt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+echo gettype($r);
+?>
+
+--CLEAN--
+<?php
+  $fileName = dirname(__FILE__)."/test.txt";
+  unlink($fileName);
+?>
+
+--EXPECTF--
```
Comment (carusogabriel) [inline ext/standard/tests/general_functions/gettype_basic.phpt:20]: Nit:
```suggestion
--EXPECT--
```

Comment (petk) [reply ext/standard/tests/general_functions/gettype_basic.phpt:20]: Hello, this one for example, I can also fix when I'll be merging it upstream. Let me know also please if you've spotted anything else...

Comment (carusogabriel) [reply ext/standard/tests/general_functions/gettype_basic.phpt:20]: @petk Should be RTM after this quick fix.

Comment (petk) [reply ext/standard/tests/general_functions/gettype_basic.phpt:20]: Awesome...

**Comment by petk** [comment]
@carusogabriel since PHP-7.2 this one returns `resource (closed)` and not the `unknown type` anymore... Do you know maybe how to test for `unknown type` more properly?

**Comment by carusogabriel** [comment]
@petk No :confused: 

Maybe @yuklia can help us on this one?

**Comment by petk** [comment]
Thank you for the pull request. Since we can't know the status of this pr anymore, I'll be merging the rest of the contributions in a single commit... More info on this soon.

**Pull request closed without merging**